src/solver/variable/economy/links/congestionProbability.h

```diff
@@ -300,16 +301,19 @@ class CongestionProbability
 
     void hourForEachLink(State& state, unsigned int numSpace)
     {
+        // Ratio take into account MC year weight
+        double ratio = (double)yearsWeight[state.year] / (double)yearsWeightSum;
```

Comment:
Why convert `yearsWeight[state.year]` and `yearsWeightSum` to `double` ?

Reply:
no reason, conversion removed.